tests/test_current_flow360_version.py

```diff
@@ -2,4 +2,4 @@
 
 
 def test_version():
-    assert __version__ == "25.7.4b0"
+    assert __version__ == "25.8.0b3"
```

Comment:
With the introduction of these new schema we need to bump "real" version. The beta version is not used anywhere (for example updated) in Python client. 
```suggestion
    assert __version__ == "25.8.1b3"
```